unit-tests/post-processing/test-post-processing-filters.py

```diff
@@ -0,0 +1,469 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#temporary fix to prevent the test from running on Win_SH_Py_DDS_CI
+#test:donotrun:dds
+
+import pyrealsense2 as rs, tempfile, os, random, csv
+from rspy import test, repo
+################################################################################################
+frame_metadata_count = 43
+
+ppf_test_cases = [
+    # All the tests below include depth-disparity domain transformation
+    # Downsample scales 2 and 3 are tested only. scales 4-7 are differ in impementation from the reference code:
```

Comment:
This comment has several misspeling mistakes (e.g. impementation) and is hard to understand. See if you can rephrase it to be more understood

Reply:
Rephrased it